### DISCUSSION TITLE
manca il pulsante supervisor

### DISCUSSION THREAD

**Discussion by 4sp3c** [indomus/forum#2243] [HUB: Home Assistant]
Ciao a tutti, ho appena installato HA su raspberry4 come Home Assistant Core dalla guida. Adesso volevo aggiungere dispositivi e mi rendo conto che mi manca il pulsante supervisor. Non c'è neppure il pulsante add-on backup e supervisor dentro impostazioni. Come riesco a risolvere? Grazie mille a tutti
![hassio](https://user-images.githubusercontent.com/95615624/210752895-acc0e9d5-65e2-4d45-b00a-2f52aba03516.JPG)

**Comment by snakuzzo** [comment]
Non risolvi, perchè non c'è alcun problema.
Non hai letto le differenze tra i vari tipi di installazione.
Con Core il supervisor non esiste (e nemmeno gli addon)

Se vuoi fare il backup delle configurazioni di core devi attivare l'integrazione `backup` nel `configuration.yaml` come indicato [QUI](https://www.home-assistant.io/integrations/backup/)

```
# Example configuration.yaml entry
backup:
```

Una volta riavviato Core avrai una voce apposita sotto `System` (come puoi vedere anche dal tuo screenshot)

**Reply by 4sp3c** [reply]
è il mio primo approccio a HA e avevo visto che era l'opzione consigliata ma mi ero perso che sul core devi poi installare manualmente eventuali add-on. Grazie mille
